app/controllers/project.php

```diff
@@ -0,0 +1,279 @@
+<?php
+
+class ProjectsController extends \BaseController {
+
+	/**
+	 * Display a listing of the resource.
+	 * GET /projects
+	 *
+	 * @return Response
+	 */
+	public function index()
+	{
+		$pTitle		= "Projects";
+
+		$counter 	=	0;
+		$user 		=	User::find(Auth::id());
+		$projects 	=	$user->projects()->get();
+		$inProjects =  $user->inProjects()->orderBy('created_at', 'desc')->take(5)->get();
+		
+		return View::make('projects.index', compact(['projects','inProjects','counter','pTitle']));
+	}
+
+	/**
+	 * Create the new project
+	 *
+	 * @return Response
+	 */
+	public function create()
+	{	
+		// Rules
+		$rules	= array('name' => 'required',);
+
+		// Create validation 
+		$validator = Validator::make( Input::all(), $rules);
+
+		if ( $validator->fails() ) {
+			return Redirect::back()->withErrors($validator)->withInput();
+		}
+		
+		$project 			=	new Project;
+		$project->name 		=	Input::get('name');
+		$project->client_id =	Input::get('client_id');
+		$project->user_id	=	Auth::id();
+		$project->save();
+
+		return Redirect::back()->with('success', Input::get('name') ." has been created.");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "success" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://172.27.59.137:9000/coding_rules#rule_key=php%3AS1192)